js/script.js

```diff
@@ -0,0 +1,263 @@
+      <p class="work-specs projectdetails">${card.description}</p>
+      <div class="for-flex projectflex2">
+        <ul>
+          <li class="flex-items item1">${card.technologies[0]}</li>
+          <li class="flex-items item2">${card.technologies[1]}</li>
+          <li class="flex-items item3">${card.technologies[2]}</li>
+        </ul>
+      </div>
+      <button type="button" id="myBtn" class="work-btn projbtn on-mobile">${card.seeProject}</button>
+    </div>
+    <!-- The Modal -->
+    <div id="myModal" class="modal">
+
+      <!-- Modal content -->
+      <div class="modal-content">
+        <div class="modal-header">
+          <span class="close">&times;</span>
+          <h2>${card.name}</h2>
+          <div class="for-flex projectflex2">
+            <ul>
+              <li class="flex-items item1">${card.technologies[0]}</li>
+              <li class="flex-items item2">${card.technologies[1]}</li>
+              <li class="flex-items item3">${card.technologies[2]}</li>
+            </ul>
+          </div>
+        </div>
+        <div class="modal-body">
+          <div><img src="./images/popup-desktop.png" class="modal-img"alt="project image"></div>
+          <div>
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+              Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+              when an unknown printer took a galley of type and scrambled it 1960s.
+            </p><br>
+            <p class="hide">
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+              Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+              when an unknown printer took a galley of type and scrambled it 1960s with the 
+              releax map lapora verita.
+            </p>
+            <div class="btns-div">
+            <a href="${card.linkLive}"><img src="./images/See Live.png" class="img-btn" alt="Live Link"></a>
+            <a href="${card.linkSource}"><img src="./images/See Source.png" class="img-btn" alt="source code"></a>
+      
+  
+      </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  
+`;
+  return ('');
+});
+
+// Get the modal
+
+const modal = document.getElementById('myModal');
+const span = document.getElementsByClassName('close')[0];
+span.onclick = () => {
+  modal.style.display = 'none';
+};
+// Get the button that opens the modal
+const proBtn = document.getElementsByClassName('projbtn');
+
+Array.from(proBtn).forEach((button) => {
+  button.addEventListener('click', () => {
+    // Perform desired action here
+    document.body.classList.toggle('no-scroll');
+    modal.style.display = 'block';
+  });
+});
+const modalClose = document.querySelector('.close');
+modalClose.addEventListener('click', () => {
+  modal.classList.remove('act');
+  document.body.classList.remove('no-scroll');
+});
```

Comment:
Great job styling the pop-up modal in the mobile view, Unfortunately, there are some elements need re-styling in the desktop view:

- [ ] Kindly increase the width and height of the pop-up project photo but keep the aspect ratio of the original image as it is.
- [ ] Please add the correct margin values to fix the spacing around and between the pop-up buttons `see live` and `see source`. and try as much as possible to stick to the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/Mike111222/MyPortfolio/assets/68796232/4bb79afa-e0f9-41f9-840b-b1a4f85e9d85)  | ![image](https://github.com/Mike111222/MyPortfolio/assets/68796232/2d07f520-b21a-44c4-aa9e-c83394a7fe26)  |

Reply:
Dear sir, I have done the changes as per the reviewer suggestion please.